examples/resources/dsm_sobject/resource.tf

```diff
@@ -32,26 +32,26 @@ resource "dsm_sobject" "sobject" {
   ]
   allowed_missing_justifications = true
   rotation_policy = {
-    interval_days = 20
-    effective_at = "20231130T183000Z"
+    interval_days          = 20
+    effective_at           = "20231130T183000Z"
     deactivate_rotated_key = true
-    rotate_copied_keys = "all_external"
-}
+    rotate_copied_keys     = "all_external"
+  }
 }
 
-// copy a security object
-/*
-Copy above security object.
-When copying a security object obj_type, key_size, allowed_key_justifications_policy, allowed_key_justifications_policy,
-allowed_missing_justifications, lms or bls should not be configured.
-*/
+## copy a security object
+
+# Copy above security object.
+# When copying a security object obj_type, key_size, allowed_key_justifications_policy, allowed_key_justifications_policy, -
+# allowed_missing_justifications, lms or bls should not be configured.
```

Comment:
Same comments here as above about the `-` and duplicated policy

Reply:
Modified.